packages/testing/src/index.ts

```diff
@@ -71,6 +78,30 @@ export class WorkflowClient extends BaseWorkflowClient {
   }
 }
 
+/**
+ * A client with the exact same API as the "normal" client with 1 exception,
+ * When this client {@link TimeSkippingWorkflowClient} waits on a Workflow's result, it will enable time skipping
+ * in the test server.
```

Comment:
```suggestion
 * A client with the exact same API as the "normal" client with one exception:
 * when `TestEnvClient.workflow` (an instance of {@link TimeSkippingWorkflowClient}) waits on a Workflow's result, it will enable time skipping
 * in the Test Server.
```